### PR TITLE
ENH: reset IJK matrix in volume information

### PR DESCRIPTION
**Pull request by riep**
IJK matrix is not reset when selecting an empty volume node

### CODE REVIEW

File: Libs/MRML/Widgets/qMRMLVolumeInfoWidget.cxx [hunk 1]
```diff
@@ -186,6 +186,15 @@ void qMRMLVolumeInfoWidget::updateWidgetFromMRML()
     double origin[3] = {0.,0.,0.};
     d->ImageOriginWidget->setCoordinates(origin);
 
+    double IJKToRASDirections[3][3] = { {1,0,0}, {0,1,0}, {0,0,1} };
```
Comment (lassoan) [inline Libs/MRML/Widgets/qMRMLVolumeInfoWidget.cxx:189]: Double values should be initialized with double (`1.0` or just `1.`). But I would rather remove this line and just use `i==j ? 1.0 : 0.0`). Also add a comment that we are setting the matrix to identity.

File: Libs/MRML/Widgets/qMRMLVolumeInfoWidget.cxx [hunk 2]
```diff
@@ -186,6 +186,15 @@ void qMRMLVolumeInfoWidget::updateWidgetFromMRML()
...
+    for (int i=0; i<3; i++)
+    {
```
Comment (lassoan) [inline Libs/MRML/Widgets/qMRMLVolumeInfoWidget.cxx:191]: I don't like Slicer's current brace indentation style, but for consistency, please use that (indent the `{` by two spaces).

**Review by lassoan — CHANGES_REQUESTED** [review]
Thank you, it looks good, just added two nitpicky comment inline (they are very easy to fix).

**Comment by riep** [comment]
corrections done

File: Libs/MRML/Widgets/qMRMLVolumeInfoWidget.cxx [hunk 3]
```diff
@@ -186,6 +186,16 @@ void qMRMLVolumeInfoWidget::updateWidgetFromMRML()
     double origin[3] = {0.,0.,0.};
     d->ImageOriginWidget->setCoordinates(origin);
 
+    // Set IJK to RAS direction matrix to identity
+    double IJKToRASDirections[3][3];
```
Comment (lassoan) [inline Libs/MRML/Widgets/qMRMLVolumeInfoWidget.cxx:190]: Is `IJKToRASDirections` still needed?

Comment (riep) [reply Libs/MRML/Widgets/qMRMLVolumeInfoWidget.cxx]: sorry, this is corrected

**Review by lassoan — APPROVED** [review]
Perfect, thank you.

**Pull request merged**